tests/UI/campaigns/functional/BO/05_customerService/03_merchandiseReturns/01_activateMerchandiseReturn.js

```diff
@@ -0,0 +1,178 @@
+      await expect(pageTitle).to.contains(merchandiseReturnsPage.pageTitle);
+    });
+
+    it(`should ${test.args.action} merchandise returns`, async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `${test.args.action}Returns`, baseContext);
+
+      const result = await merchandiseReturnsPage.setOrderReturnStatus(page, test.args.enable);
+      await expect(result).to.contains(merchandiseReturnsPage.successfulUpdateMessage);
+    });
+
+    it('should go to orders page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `goToOrdersPage${index}`, baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.ordersParentLink,
+        dashboardPage.ordersLink,
+      );
+
+      const pageTitle = await ordersPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(ordersPage.pageTitle);
+    });
+
+    it('should filter the Orders table by the default customer and check the result', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `filterOrder${index}`, baseContext);
+
+      await ordersPage.filterOrders(page, 'input', 'customer', DefaultAccount.lastName);
+
+      const textColumn = await ordersPage.getTextColumn(page, 'customer', 1);
+      await expect(textColumn).to.contains(DefaultAccount.lastName);
+    });
+
+    it('should go to the first order page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `goToOrderPage${index}`, baseContext);
+
+      // View order
+      await ordersPage.goToOrder(page, 1);
+
+      const pageTitle = await viewOrderPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(viewOrderPage.pageTitle);
+    });
+
+    it(`should change the order status to '${Statuses.shipped.status}' and check it`, async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `updateOrderStatus${index}`, baseContext);
+
+      const result = await viewOrderPage.modifyOrderStatus(page, Statuses.shipped.status);
+      await expect(result).to.equal(Statuses.shipped.status);
+    });
+
+    it('should check that the button \'Return products\' is visible', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `checkReturnProductsButton${index}`, baseContext);
+
+      const result = await viewOrderPage.isReturnProductsButtonVisible(page);
+      await expect(result).to.equal(test.args.enable);
+    });
+
+    it('should go to FO and login', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `openTheShop${index}`, baseContext);
+
+      // Click on view my shop
+      page = await viewOrderPage.viewMyShop(page);
+
+      // Change FO language
+      await homePage.changeLanguage(page, 'en');
+
+      const isHomePage = await homePage.isHomePage(page);
+      await expect(isHomePage, 'Home page is not displayed').to.be.true;
+
+      if (index === 0) {
```

Comment:
```suggestion
      if (test.args.enable) {
```
It's more obvious than using an index you don't understand :sweat_smile: